### ISSUE TITLE
What about a study using Chester in a fild hospital in Uganda?

### ISSUE THREAD

**Issue by Mactheboss78** [mlmed/chester-xray#27]
Hi,
I was not able to find contacts online for the project.
We support a hospital in the north of Uganda and they have no radiologist.
Would it be ok to prepare (and have ethic clearance) for a project using both Chester and teleradiology?
Do I need to get permission from one of the authors?
Thanks,
AC

**Comment by ieee8023** [comment]
Yes it is fine with me if you want to do research with it. Let me know if you have any questions. I cannot guarantee any support for the tool.
